enroll/fields.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+DAY_OF_THE_WEEK = {
+    '1': 'Monday',
```

Comment:
Consider starting indexing with 0.

Reply:
Django docs use enumeration types with 1 as the lowest key, e.g. [here](https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types) (search via IntegerChoices).

Reply:
Cringe